content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.setUTCMinutes()'
+Desciption: 'brief, description of the setUTCMinutes function, examples'
+Subjects: 
+    - 'Computer Science' 
+    - 'Web Development'
+Tags: 
+    - 'Methods'
+    - 'Date'
+CatalogContent:
+    - 'introduction-to-javascript'
+    - 'paths/front-end-engineer-career-path'
+---
+
+The **.setUTCMinutes()** method of [Date](https://www.codecademy.com/resources/docs/javascript/dates) instances changes the minutes for this instance according to **universal time (UTC)**.
```

Comment:
```suggestion
The **`.setUTCMinutes()`** method of `Date` instances changes the minutes for this instance according to universal time (UTC).
```
We generally don't link out to parent entries because the Docs navigation provides them automatically.

Reply:
I missed this detail of the Docs capability. Thank you.